content/en/docs/contribute/participate/roles-and-responsibilties.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Roles and responsibilities
+content_type: concept
+weight: 10
+---
+
+<!-- overview -->
+
+Anyone can contribute to Kubernetes. As your contributions to SIG Docs grow, you can apply for different levels of membership in the community. These roles allow you to take on more responsibility within the community. Each role requires more time and commitment. The roles are:
```

Comment:
Governance is nonhierarchical, and there are no physical differences in elevation.
```suggestion
Anyone can contribute to Kubernetes. As your contributions to SIG Docs grow, you can apply for different membership roles in the community. These roles allow you to take on more responsibility within the community. Each role requires more time and commitment. The roles are:
```